enhancements/ingress/gateway-api-with-cluster-ingress-operator.md

```diff
@@ -295,6 +288,133 @@ configures Envoy accordingly.  Configuring a load balancer and DNS enables
 clients to connect through Envoy to backend applications as specified by the
 Gateway and HTTPRoute objects.
 
+Following is the ServiceMeshControlPlane CR that the Ingress Operator creates:
+
+```yaml
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  name: gwapi
+  namespace: openshift-ingress
+spec:
```

Comment:
Would it be smart to turn on access logging for dev preview? I feel like that might be quite helpful and we know folks aren't supposed to use it for production, right?
```
  proxy:
    accessLogging:
      envoyService:
        enabled: true
      file:
        name: /dev/stdout
```

Reply:
That seems reasonable.  Added.